grpclog/loggerv2_test.go

```diff
@@ -61,3 +66,264 @@ func checkLogForSeverity(s int, b []byte) error {
 	}
 	return nil
 }
+
+// makeLogFuncErrr returns a logger printFunc which generates a test error,
+// for testing that the log func is not called
+func makeLogFuncErr(t *testing.T) func() string {
```

Comment:
can we move all the helper functions above the tests?

Reply:
Changed